config/locales/devise/en.yml

```diff
@@ -5,22 +5,22 @@ en:
       already_confirmed: Your email address has already been confirmed. %{action}
       confirmed: You have confirmed your email address
       confirmed_but_must_set_password: You have confirmed your email address
-      send_instructions: >
-        You will receive an email with instructions for how to confirm your
-        email address in a few minutes.
-      send_paranoid_instructions: >
-        You will receive an email with instructions for how to confirm your email
-        address in a few minutes.
+      send_instructions: 'You will receive an email with instructions for how to confirm your email address in a few minutes.
+
+'
```

Comment:
It looks like this introduced a bunch of additional spaces before the end quote. I see several instances where this happened in this PR. Was that intentional?